lib/ex_huobi/futures/util.ex

```diff
@@ -0,0 +1,27 @@
+defmodule ExHuobi.Futures.Util do
```

Comment:
I know naming is difficult. But in this case, `Util` not the best. can we rename to `ExHuobi.Futures.Handler`